### PR TITLE
fix having no forecasts, using empty

### PR DESCRIPTION
**Pull request by xmariachi**
## What is the purpose of the change

Fix having no forecasts, nonce must be closed anyway

## Testing and Verifying

This change is covered by related tests

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?

Where is the change documented?
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Allora documentation site `docs.allora.network` source code at: `https://github.com/allora-network/docs`
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/emissions/keeper/actor_utils/worker.go [hunk 1]
```diff
@@ -67,7 +69,13 @@ func CloseWorkerNonce(k *keeper.Keeper, ctx sdk.Context, topicId keeper.TopicId,
 	// Get all forecasts from this topicId, nonce
 	forecasts, err := k.GetForecastsAtBlock(ctx, topicId, nonce.BlockHeight)
 	if err != nil {
-		return err
+		if errors.Is(err, collections.ErrNotFound) {
```
Comment (kpeluso) [inline x/emissions/keeper/actor_utils/worker.go:72]: should this not go in `GetForecastsAtBlock()` just like `GetInferencesAtBlock`...

**Review by kpeluso — CHANGES_REQUESTED** [review]

File: x/emissions/keeper/actor_utils/worker.go (hunk 1 above)
Comment (xmariachi) [reply x/emissions/keeper/actor_utils/worker.go:72]: I'm good moving it in there, though for sake of all other clients of this code alredy using it like this, i reckon for now is ok. But I'm ok to change it like that and fix the other places (like i did in inferences)

Comment (xmariachi) [reply x/emissions/keeper/actor_utils/worker.go:72]: nah, quickly fixing

Comment (xmariachi) [reply x/emissions/keeper/actor_utils/worker.go]: fixed

**Review by kpeluso — APPROVED** [review]

**Pull request merged**